rootfs/opt/ibm/router/nginx/template/nginx.tmpl

```diff
@@ -215,7 +215,7 @@ http {
         add_header X-Frame-Options "SAMEORIGIN";
         add_header X-Content-Type-Options nosniff;
         add_header X-XSS-Protection "1; mode=block";
-        add_header Strict-Transport-Security "max-age=31536000; includeSubDomains";
+        add_header Strict-Transport-Security "max-age=63072000; includeSubDomains";
```

Comment:
@subbarao-meduri is there an issue associated with this change? If so, we might want to link the PR to that issue.

Reply:
@dislbenn this is forward port of the fix https://github.com/stolostron/management-ingress/pull/72 that went into release-2.5 stream